fraud/service_test.go

```diff
@@ -21,8 +21,8 @@ import (
 
 func TestService_Subscribe(t *testing.T) {
 	s, _ := createService(t, false)
-	require.NoError(t, s.RegisterProofs(mockProofType))
 	proof := newValidProof()
+	require.NoError(t, s.Start(context.TODO()))
```

Comment:
let's use a ctx that will be cancelled after test finishes since that's the pattern we use everywhere else

Reply:
done